README.md

```diff
@@ -9,7 +8,186 @@ NestJS-Matic
 
 Matic PoS and Plasma utilities for NestJS based on [matic.js](https://github.com/maticnetwork/matic.js/)
 
-> Currently under developement 🤓. If you want to help let me know!
+## Install
+
+```sh
+npm i nestjs-matic
+```
+
+## Register module
+
+### Zero configuration
+
+Import the `MaticModule` to your module and pass in it through your matic object options, just like that:
+
+```ts
+import { MaticModule } from 'nestjs-matic';
+
+@Module({
+  imports: [MaticModule.forRoot(MATIC_OPTIONS)],
+  ...
+})
+class MyModule {}
+```
+
+**NOTE:** we describe the [MATIC_OPTIONS](#configuration-params) down below.
```

Comment:
I think here you need to put something like:  By default MaticModule will try to connect using the MaticPlama client.

Reply:
Ok, I will fix this out :+1: